src/bun.js/webcore/response.zig

```diff
@@ -688,6 +688,9 @@ pub const Fetch = struct {
         }
 
         pub fn onReject(this: *FetchTasklet) JSValue {
+            if (this.result.exception) |exception| {
```

Comment:
Does the memory still get freed?

Reply:
The code is pretty similar to toErrorInstance from SystemError, I moved the creation of the exception to onReject (the default fail SystemError already is here) so I think we don't have any memory leaks here